pages/dev/send-tokens/overview.mdx

```diff
@@ -44,7 +44,9 @@ function approve(address spender, uint256 amount) external returns (bool);
 
 Here, `spender` is the Gateway address on the source chain.
 
-Find a list of assets, their names and their addresses: [Mainnet](../reference/mainnet-contract-addresses) | [Testnet](../reference/testnet-contract-addresses).
+Find a list of assets, their names and their addresses: [Mainnet](../reference/mainnet-contract-addresses) | [Testnet](../reference/testnet-contract-addresses). Note that for the same asset denom,
+the ERC20 symbol could be different on different chains.
+For e.g., `USDC` on Ethereum is linked to `axlUSDC` on Avalanche.
```

Comment:
The correct english format  for eg is to use this as a parenthetical statement attached to the previous statement.
"the ERC20 symbol could be different on different chains (e.g. USDC on ethereum..."
or you could just use words in your separate sentence.
"For example, `USDC` on ethereum is linked..."